### PR TITLE
[AIFlow] Introduces the command line interfaces of the workflow and job

### LINKED ISSUE #173: [AIFlow] Introduces the command line interfaces of the workflow and job
### Describe the feature
The workflow operations need to support the command line interfaces for maintenance.

### PR DESCRIPTION
**Pull request by SteNicholas**
The workflow operations need to support the command line interfaces for maintenance. Fix #173

### CODE REVIEW

File: ai_flow/cli/cli_parser.py [hunk 1]
```diff
@@ -149,10 +166,112 @@ class GroupCommand(NamedTuple):
     )
 )
 
+WORKFLOW_COMMANDS = (
+    ActionCommand(
+        name='delete',
+        help='Deletes all DB records related to the specified workflow.',
```
Comment (jiangxin369) [inline ai_flow/cli/cli_parser.py:172]:
```suggestion
        help='Delete all DB records related to the workflow.',
```

File: ai_flow/cli/cli_parser.py [hunk 2]
```diff
@@ -149,10 +166,112 @@ class GroupCommand(NamedTuple):
...
+    ),
+    ActionCommand(
+        name='list',
+        help='Lists all the workflows.',
+        func=lazy_load_command('ai_flow.cli.commands.workflow_command.workflow_list'),
+        args=(ARG_PROJECT_PATH, ARG_OUTPUT),
+    ),
+    ActionCommand(
+        name='list-executions',
+        help='Lists the workflow executions given a workflow name.',
```
Comment (jiangxin369) [inline ai_flow/cli/cli_parser.py:184]: List all workflow executions of the workflow.

File: ai_flow/cli/commands/workflow_command.py [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+            'workflow_name': x.workflow_name,
+            'properties': x.properties,
+            'scheduling_rules': x.scheduling_rules
+        },
+    )
+
+
+@init_config
+def workflow_show_execution(args):
+    """SShows the workflow execution given a workflow execution id."""
```
Comment (jiangxin369) [inline ai_flow/cli/commands/workflow_command.py:110]:
```suggestion
    """Shows the workflow execution by workflow execution id."""
```

File: ai_flow/cli/commands/workflow_command.py [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+@init_config
+def workflow_stop_execution(args):
+    """Stops the workflow execution given a workflow execution id."""
+    workflow_execution = stop_workflow_execution(args.workflow_execution_id)
+    print("Workflow Execution: {}, stopped: {}.".format(args.workflow_execution_id, workflow_execution is not None))
+
+
+@init_config
+def workflow_stop_executions(args):
+    """Stops the workflow executions given a workflow name."""
```
Comment (jiangxin369) [inline ai_flow/cli/commands/workflow_command.py:143]:
```suggestion
    """Stops all workflow executions given a workflow name."""
```

File: ai_flow/test/cli/commands/ut_workflows/workflows/test_command/test_command.py [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+_SQLITE_DB_URI = '%s%s' % ('sqlite:///', _SQLITE_DB_FILE)
+_PORT = '50051'
+_SERVER_URI = 'localhost:{}'.format(_PORT)
+_SCHEDULER_CLASS = 'ai_flow.test.api.mock_plugins.MockScheduler'
+_WORKFLOW_NAME = 'test_workflow_operation'
+
+PROJECT_PATH = get_parent_dir(get_parent_dir(get_file_dir(__file__)))
+
+
+class TestCommand(unittest.TestCase):
```
Comment (jiangxin369) [inline ai_flow/test/cli/commands/ut_workflows/workflows/test_command/test_command.py:41]: Seems no test case in this file.

Comment (SteNicholas) [reply ai_flow/test/cli/commands/ut_workflows/workflows/test_command/test_command.py:41]: This is the basic test class for the workflow and job command.

**Review by jiangxin369 — APPROVED** [review]

**Pull request merged**